### ISSUE TITLE
Detupling missing when calling a module function value

### ISSUE THREAD

**Issue by Jand42** [fsharp/FSharp.Compiler.Service#495]
If I have
```fsharp
module LetLambda =
    let f =
        () // so that it is a function value
        fun a b -> a + b
```
then use this in `[ 1, 2 ] |> List.map (fun (a, b) -> LetLambda.f a b)`, the lambda is exposed by FCS as `fun tupledArg -> (LetLambda.f () a) b`. Usual detupling is missing, so variables `a` and `b` are used but not defined with a `Let`.

**Comment by dsyme** [comment]
Fixed by https://github.com/fsharp/FSharp.Compiler.Service/pull/549

**Comment by dsyme** [comment]
Fix is in 2.0.0.7
